src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurity.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurity
+    {
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurity_ReleaseBuffer")]
```

Comment:
The naming convention for entrypoint prefix is to match the library name: `Libraries.NetSecurityNative` -> `EntryPoint="NetSecurityNative_XXX`